lib/Controller/PageController.php

```diff
@@ -405,6 +425,63 @@ protected function guestEnterRoom(string $token, string $password): Response {
 		return $response;
 	}
 
+	/**
+	 * @return TemplateResponse|RedirectResponse
+	 * @throws HintException
+	 */
+	protected function invitedEmail(
+		string $token,
+		string $email,
+		#[SensitiveParameter]
+		string $accessToken,
+	): Response {
+		try {
+			$actorId = hash('sha256', $email);
+			$this->manager->getRoomByAccessToken(
+				$token,
+				Attendee::ACTOR_EMAILS,
+				$actorId,
+				$accessToken,
+			);
+			$this->talkSession->renewSessionId();
+			$this->talkSession->setAuthedEmailActorIdForRoom($token, $actorId);
+		} catch (RoomNotFoundException) {
```

Comment:
Banned emails are also redirected as 404 to login page, that's what we want for now?

Reply:
If you got banned, yes.